File: src/main/java/com/anur/exception/HanabiException.java
package com.anur.exception;

/**
 * Created by Anur IjuoKaruKas on 2019/1/21
 */
public class HanabiException extends RuntimeException {

    public HanabiException() {
        super();
    }

    public HanabiException(String message) {
        super(message);
    }

    public HanabiException(String message, Throwable cause) {
        super(message, cause);
    }

    public HanabiException(Throwable cause) {
        super(cause);
    }

    protected HanabiException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: src/main/java/com/anur/exception/HanabiException.kt
package com.anur.exception


/**
 * Created by Anur IjuoKaruKas on 2019/7/5
 */
open class HanabiException : RuntimeException {
    constructor(msg: String) : super(msg)
    constructor(msg: String, thr: Throwable) : super(msg, thr)
    constructor(thr: Throwable) : super(thr)
}